src/renderer/vdom/h.ts

```diff
@@ -14,7 +14,7 @@ const stack: any[] = [];
 
 export function h(nodeName: string | d.FunctionalComponent<d.PropsType>, vnodeData: d.PropsType, child?: d.ChildType): d.VNode;
 export function h(nodeName: string | d.FunctionalComponent<d.PropsType>, vnodeData: d.PropsType, ...children: d.ChildType[]): d.VNode;
-export function h(nodeName: any, vnodeData: any, child?: any) {
```

Comment:
This function, uses arguments, causing a mutation in an arguments causes complete deoptimization, since `arguments` needs to be in sync with all the arguments at any given time.